built-ins.c

```diff
@@ -30,7 +30,51 @@ int handle_builtins(char **args, char **env)
 			print_env(env);
 		return (1);
 	}
+	if (_strcmp(args[0], "cd"))
+	{
+		handleCD(args + 1, env);
+		return (1);
+	}
 
 	return (0);
 }
 
+/**
+ * handleCD - handles the execution of cd command
+ * @args: array of cd args
+ * @env: environment array
+ * Return: void
+ */
+void handleCD(char **args, char **env)
+{
+	if (args[0] == NULL)
```

Comment:
if the first arg is `NULL` shouldn't you just print an error and return

Reply:
no that means `ls` with no args has been supplied

Reply:
which means that you have to write an error and quit the function with an empty return.
you mean `cd` right? not `ls`.

Reply:
no `cd` is a short cut for `cd $HOME`